go-build-repo-mod.sh

```diff
@@ -16,11 +18,11 @@ errCode=0
 # Assume parent folder of go.mod is module root folder
 #
 for sub in $(find . -name go.mod -not -path '*/vendor/*' | xargs -n1 dirname | sort -u) ; do
-	pushd "${sub}" >/dev/null
+	pushd "${sub}" > ${outfile}
```

Comment:
For `pushd` & `popd`, we truly do not care about the output, we just need to prevent it from being displayed to the user.

Q: Does the system you run this on (is it WSL?) have **any** canonical technique for discarding script output?